### PR TITLE
Fix aliens not understanding all surgeries

### PR DESCRIPTION
**Pull request by Cenrus**
Fixes the abductor scientist not understanding surgery.

:cl:
fix: Abductor scientists now understand all surgery types.
/:cl:

### CODE REVIEW

**Comment by ItsTisme** [comment]
was about to say that I made a PR similair like this but you got it good so its fine

**Comment by utilbird** [comment]
please write a description and CL, even if it is short

File: code/modules/surgery/surgery.dm [hunk 1]
```diff
@@ -43,7 +43,7 @@
 	if(replaced_by == /datum/surgery)
 		return FALSE
 
-	if(HAS_TRAIT(user, TRAIT_SURGEON))
+	if(user.mind && HAS_TRAIT(user.mind, TRAIT_SURGEON))
```
Comment (park66665) [inline code/modules/surgery/surgery.dm]:
```suggestion
	if(HAS_TRAIT(user, TRAIT_SURGEON) || (user.mind && HAS_TRAIT(user.mind, TRAIT_SURGEON)))
```

**Review by park66665 — APPROVED** [review]
All clear. This PR is good to go.

**Pull request merged**